packages/peregrine/src/hooks/useRestApi.js

```diff
@@ -32,7 +32,7 @@ export const useRestApi = endpoint => {
                 receiveError(error.baseMessage);
             }
         },
-        [receiveError, receiveResponse, setLoading]
+        [endpoint, receiveError, receiveResponse, setLoading]
```

Comment:
This change has nothing to do with this PR but it failed linting and I couldn't push without it.

Reply:
*cough* `--no-verify` *cough* I mean what? Good job thanks!